js/modules/k6/experimental/README.md

```diff
@@ -0,0 +1,224 @@
+# Experimental Modules
+
+This folder contains experimental modules and their documentation. Some of the experimental functionalities' code lives in this folder itself, other like the experimental redis, or experimental websockets modules are imported and injected in k6 directly from their respective repository's go modules. 
+
+## Redis
```

Comment:
I would prefer making this smaller. 

IMO there is no point in copy-pasting the Redis README here. We will likely need to drop it in the future so we are just doubling our work. And if we update it we will need to update it here or forget about it.

Maybe just list the current experiments :shrug: maybe with list to the repos?

Or maybe just point  to initcontext.go and tell people to figure it out through go modules :shrug: With a comment that the code doesn't live in the k6 repo(we will gloss over the existance of `./vendor` ;))

In any case - the above section also needs updates ;)

Reply:
Will do that:
- Remove the duplicate documentation (this was written before the docs)
- Point to the extensions repos
- Point to the modules' documentation on the docs website